bciers/libs/components/src/form/SingleStepTaskListForm.tsx

```diff
@@ -79,77 +75,78 @@ const SingleStepTaskListForm = ({
 
   const formRef = createRef<any>();
 
-  useEffect(() => {
-    // Do not validate on component mount if there is no formData
-    if (!formData || Object.keys(formData).length === 0 || !formRef?.current)
-      return;
-
-    // If there is formData, validate the form sections and
-    // set the formSectionStatus on component mount
-    const validator = formRef?.current.state.schemaUtils.getValidator();
-    const errorData = validator.validateFormData(formData, schema);
-    const errorSchema = errorData.errorSchema;
-
-    const formSectionErrorList = formSectionList.reduce(
-      (acc: { [key: string]: boolean }, section: string) => {
-        const sectionErrors = errorSchema?.[section] ?? {};
-        const isValid = Object.keys(sectionErrors).length === 0;
-        acc[section] = isValid;
-        return acc;
-      },
-      {},
-    );
-
-    setIsLiveValidate(true);
-    setFormSectionStatus(formSectionErrorList);
-  }, []);
+  // useEffect(() => {
```

Comment:
I think we need a prop to disable validation/the green checkmark. Sepehr is still using this for the contact form.

Reply:
I wonder if it will confuse users if some of the forms have checkmarks and others don't. @suhafa , thoughts?

Reply:
@suhafa  said if implementing checkmarks will take more than 2-3 sprints, we should probably remove it from everywhere. I made a task ticket to look at it in the future: https://github.com/orgs/bcgov/projects/122/views/2?filterQuery=&pane=issue&itemId=72123051